plugin/code_actions.py

```diff
@@ -293,11 +292,11 @@ def run(self, edit: sublime.Edit, event: Optional[dict] = None,
             actions_manager.request_for_region_async(
                 view, covering, session_buffer_diagnostics, self.handle_responses_async, dict_kinds)
 
-    def handle_responses_async(self, responses: CodeActionsByConfigName = None) -> None:
-        if responses:
-            self.commands_by_config = responses
+    def handle_responses_async(self, responses: CodeActionsByConfigName,
+                               has_commands_by_config: bool = False) -> None:
```

Comment:
Single line is fine if it fits within 120 column mark. Also renamed to "*_actions_*".
```
    def handle_responses_async(self, responses: CodeActionsByConfigName, has_actions_by_config: bool = False) -> None:
```

Reply:
> actions

Thanks for your advice. Code specification is very important